heft-plugins/heft-webpack4-plugin/src/Webpack4Plugin.ts

```diff
@@ -53,20 +58,15 @@ type ExtendedMultiCompiler = TWebpack.MultiCompiler & {
 };
 
 export interface IWebpackPluginOptions {
-  devConfigurationPath: string | undefined;
-  configurationPath: string | undefined;
+  devConfigurationPath?: string | undefined;
+  configurationPath?: string | undefined;
```

Comment:
```suggestion
  devConfigurationPath?: string;
  configurationPath?: string;
```

Reply:
Doesn't hurt to double specify. Semantically `x: string | undefined` and `x?: string` mean slightly different things. Technically `{ x: undefined }` does not satisfy the latter, while `{}` does not satisfy the former.